pallets/gear/src/benchmarking.rs

```diff
@@ -16,27 +16,213 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-//! Benchmarking setup for pallet-template
+//! Gear pallet benchmarking
+
+#![cfg(feature = "runtime-benchmarks")]
 
 use super::*;
+use codec::Encode;
+use common::{IntermediateMessage, Origin};
+use parity_wasm::elements::{
```

Comment:
Philosophic question :-)
Isn't better to use `use parity_wasm::elements::*;` when we import so many subjects?

Reply:
Agree, it'd be more readable that way.